kolibri/plugins/coach/assets/src/views/common/QuizLessonDetailsHeader.vue

```diff
@@ -72,6 +96,18 @@
       resource() {
         return this.examOrLesson === 'lesson' ? this.lesson : this.exam;
       },
+      isActive() {
+        return this.exam.active;
+      },
+      isExamDraft() {
+        return this.exam.draft;
+      },
+      isOldDataModelVersion() {
```

Comment:
Marcella I think mentioned this too. I think this name could be a little confusing depending on who reads it. From what I can understand, all we are doing here is verifying that the current exam is of an older version? I think names like `isExamOldVersion` could be explicit yet easy to understand.